src/bytes.rs

```diff
@@ -818,8 +795,7 @@ impl<S> SymbolTable<S> {
     #[must_use]
     fn alloc(contents: Vec<u8>) -> Slice {
         let boxed_slice = contents.into_boxed_slice();
```

Comment:
going from `Vec<u8>` to `BString` would require activating the `std` feature.

I'd like to avoid doing that. One way we can work around this is by manually implementing `Debug` and calling `as_ref().as_bstr()` on the `Owned` boxed slice.